### DISCUSSION TITLE
How to run the code in tests file

### DISCUSSION THREAD

**Discussion by VivianChan99** [Telecominfraproject/oopt-gnpy#429] [Q&A] [answered]
Hello, when I ran the program code in the tests file, the following problems appeared. Please help answer, thank you.

**Question One:** A ''ImportError while loading conftest'' error occurred.
**Question Two:** A ''ModuleNotFoundError'' error occurred.

Display the contents as shown in the following image. Click on the link below to view the picture.
[https://user-images.githubusercontent.com/100559062/155953832-4fdd0623-b719-4e9b-bbb5-ffb35f645940.png](url)

Looking forward to your reply. Thank you.

**Comment by jktjkt** [comment, marked as answer]
Hello, please follow the [documentation on installing GNPy](https://gnpy.readthedocs.io/en/master/install.html). If the problem persists, please let us know (the picture you linked to won't open here for me).

**Reply by VivianChan99** [reply]
Thanks for your reply. I will try to install GNPy.
